src/components/SetupComponents/SetupMarketsBlock/SetupMarketsBlock.pcss

```
@@ -20,8 +20,8 @@
     select {
       border: 2px solid var(--secondary-contrast-cta-color-level-1);
       border-radius: 4px;
-      max-height: 500px;
-      min-height: 500px;
+      max-height: calc(100ch - 144px - 42px - 85px - 45px);
```

1. Maybe `vh` instead of `ch`?
```suggestion
      max-height: calc(100vh - 316px);
```